oak_restricted_kernel_runtime/src/channel.rs

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use anyhow::anyhow;
+use oak_channel::{Read, Write};
+
+pub const OAK_CHANNEL_FD: i32 = 10;
```

Comment:
I personally like `descriptor` because using Linux-like terminology makes it much more understandable. Since we already have a ton of new concepts for a new person to learn.

Reply:
I think the Fuchsia "handle" means something semantically different than the Linux-y file descriptors, so I'm inclined to keep the terminology here.